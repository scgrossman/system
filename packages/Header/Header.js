import React from "react";

import styles from "./Header.scss";

const Header = ({ children }) => {
  return <h1 className={styles.header}>{children}</h1>;
};

export default Header;
