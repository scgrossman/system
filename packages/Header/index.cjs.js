require("./dist/index.css");
const Header = require("./dist/index.cjs");

module.exports = Header;
