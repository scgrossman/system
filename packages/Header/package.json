{
  "name": "@rdmsystem/header",
  "version": "1.0.2",
  "description": "A test header component",
  "main": "index.cjs.js",
  "module": "index.es.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "prop-types": "^15.6.2"
  }
}
