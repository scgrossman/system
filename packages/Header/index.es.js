import "./dist/index.css";
import Header from "./dist/index.es";

export default Header;
