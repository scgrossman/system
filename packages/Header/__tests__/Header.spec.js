import React from "react";
import { shallow } from "enzyme";

import Header from "../Header";

describe("Box", () => {
  const doShallow = props => shallow(<Header {...props}>Some content</Header>);

  it("renders", () => {
    const box = doShallow();

    expect(box).toMatchSnapshot();
  });
});
