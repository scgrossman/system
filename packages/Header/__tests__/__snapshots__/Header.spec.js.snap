// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box renders 1`] = `
<h1
  className="header"
>
  Some content
</h1>
`;
